src/Controller/PageController.php

```diff
@@ -275,4 +276,49 @@ public function healthcheck()
 
         return new JsonResponse(['message' => 'OK']);
     }
+
+    public function statuscheck()
+    {
+        $client = HttpClient::create(['verify_peer' => false, 'verify_host' => false, 'timeout' => 5]);
```

Comment:
'verify_peer' and 'verify_host' what do these mean exactly? Just curious.

Reply:
Remember the SSL issue I mentioned during stand up, this bypass that issue.

Reply:
Ah ok thanks for clarifying that.